build.sh

```diff
@@ -41,15 +41,20 @@ BUILDCONFIG="opinicus"
 # Setup internal variables
 KCONFIG="${CWD}/configs/${BUILDCONFIG}_config"
 KERNEL_BUILD_DIR="${CWD}/_build_armhf/${BUILDCONFIG}-linux"
+KERNEL_IMAGE="uImage-sun7i-a20-opinicus_v1"
 BUILD_OUTPUT_DIR="${CWD}/_build_armhf/"
 DEBIAN_DIR="${BUILD_OUTPUT_DIR}/debian"
 BOOT_FILE_OUTPUT_DIR="${DEBIAN_DIR}/boot"
 
 INITRAMFS_MODULES_REQUIRED="sunxi_wdt.ko ssd1307fb.ko drm.ko sun4i-backend.ko sun4i-drm.ko sun4i-tcon.ko sun4i-drm-hdmi.ko sun4i-hdmi-i2c.ko"
+
```

Comment:
All this modules are needed?? Including ssd1307fb???

Reply:
Since support is dropped for UM3, the ssd1307fb can be dropped.